.github/workflows/ci.yaml

```diff
@@ -8,115 +8,106 @@ on:
 
   build:
     runs-on: ubuntu-22.04
+    needs: setup
     steps:
-    - uses: actions/checkout@v4
-    - uses: Atsumi3/actions-setup-fvm@0.0.3
-    - uses: bluefireteam/melos-action@v3
-    - run: sudo apt update
-    - run: sudo apt install -y clang cmake curl libgtk-3-dev ninja-build pkg-config unzip xvfb
-      env:
-        DEBIAN_FRONTEND: noninteractive
-    - run: melos build
+      - uses: actions/checkout@v4
+      - uses: Atsumi3/actions-setup-fvm@0.0.3
+      - uses: bluefireteam/melos-action@v3
+      - run: sudo apt update
+      - run: sudo apt install -y clang cmake curl libgtk-3-dev ninja-build pkg-config unzip xvfb
+        env:
+          DEBIAN_FRONTEND: noninteractive
+      - run: melos build
 
-  coverage:
+  analyze:
     runs-on: ubuntu-22.04
+    needs: setup
     steps:
-    - uses: actions/checkout@v4
-    - uses: Atsumi3/actions-setup-fvm@0.0.3
-    - uses: bluefireteam/melos-action@v3
-    - run: sudo apt update
-    - run: sudo apt install -y lcov
-      env:
-        DEBIAN_FRONTEND: noninteractive
-    - run: melos run coverage
-    - uses: codecov/codecov-action@v4
-      with:
-        token: ${{secrets.CODECOV_TOKEN}}
+      - uses: actions/checkout@v3
+      - name: Restore repository cache
+        uses: actions/cache@v3
+        with:
+          path: .
+          key: ${{ runner.os }}-repo-${{ github.sha }}-generated
+          restore-keys: |
+            ${{ runner.os }}-repo-
+      - name: Restore repository cache
+        uses: actions/cache@v3
+        with:
+          path: .
+          key: ${{ runner.os }}-repo-${{ github.sha }}-generated
+          restore-keys: |
+            ${{ runner.os }}-repo-
+      - uses: Atsumi3/actions-setup-fvm@0.0.3
+      - uses: bluefireteam/melos-action@v2
+      - run: melos analyze --fatal-infos
 
   format:
     runs-on: ubuntu-22.04
+    needs: setup
     steps:
-    - uses: actions/checkout@v4
-    - uses: Atsumi3/actions-setup-fvm@0.0.3
-    - uses: bluefireteam/melos-action@v3
-    - run: melos run format
-
-  integration:
-    runs-on: ubuntu-22.04
-    steps:
-    - uses: actions/checkout@v4
-    - uses: Atsumi3/actions-setup-fvm@0.0.3
-    - uses: bluefireteam/melos-action@v3
-    - run: sudo apt update
-    - run: sudo apt install -y clang cmake curl libgtk-3-dev ninja-build pkg-config unzip xvfb
-      env:
-        DEBIAN_FRONTEND: noninteractive
-    - run: melos exec -c 1 --dir-exists=integration_test -- \
-        xvfb-run -a xvfb-run -a -s '-screen 0 1024x768x24 +extension GLX' \
-            flutter test integration_test
+      - uses: actions/checkout@v3
+      - name: Restore repository cache
+        uses: actions/cache@v3
+        with:
+          path: .
+          key: ${{ runner.os }}-repo-${{ github.sha }}-generated
+          restore-keys: |
+            ${{ runner.os }}-repo-
+      - uses: Atsumi3/actions-setup-fvm@0.0.3
+      - uses: bluefireteam/melos-action@v2
+      - run: melos format:exclude
 
```

Comment:
It seems like the integration test went missing :)

Reply:
There aren't any integration tests in this repo :smile:

Reply:
Oh wow.. why did we have that job then? :D

Reply:
Probably copy pasted from some other repo I'm guessing (I wouldn't be too surprised if I was the one that did it...) :sweat_smile: